pom.xml

```diff
@@ -907,6 +907,11 @@
                 <artifactId>flink-csv</artifactId>
                 <version>${flink.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-orc_${scala.binary.version}</artifactId>
```

Comment:
please add dependency in LICENSE and NOTICE

Reply:
Added